src/movies.js

```diff
@@ -1,23 +1,70 @@
 // The `movies` array from the file `src/data.js`.
-console.log('movies: ', movies);
+//console.log('movies: ', movies);
 
+//const movies = require("./data");
 
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors() {}
+function getAllDirectors(movies) {
+  const allDirectors = movies.map(function (movie) {
+    return movie.director;
+  });
+  return allDirectors;
+}
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-function howManyMovies() {}
+function howManyMovies(movies) {
+  const spielbergMovies = movies.filter(function (movie) {
+    if (
+      movie.director === 'Steven Spielberg' &&
+      movie.genre.includes('Drama') === true
+    ) {
+      return true;
+    }
+  });
+  let counter = 0;
+  for (let i = 0; i < spielbergMovies.length; i++) {
+    counter += 1;
+  }
+  return counter;
```

Comment:
This code is actually unnessary. Remember, you already have array with filtered Spielberg movies. So we could just count how many movies are in that array -> return `spielbergMovies.length`